apps/ui/src/App.vue

```diff
@@ -3,14 +3,19 @@ import { startIntercom } from './helpers/intercom';
 import { Transaction } from './types';
 
 const el = ref(null);
+const sidebarSwipeEnabled = ref(true);
 
 const route = useRoute();
 const router = useRouter();
 const uiStore = useUiStore();
 const { modalOpen } = useModal();
 const { init, app } = useApp();
 const { web3 } = useWeb3();
-const { isSwiping, direction } = useSwipe(el);
+const { isSwiping, direction } = useSwipe(el, {
+  onSwipe(e: TouchEvent) {
+    sidebarSwipeEnabled.value = !e.target?.closest('[data-no-sidebar-swipe]');
```

Comment:
Great idea, maybe we can enable sidebar swipe once user reaches far left? 
```suggestion
    const noSideBarSwipe = (e.target as Element)?.closest(
      '[data-no-sidebar-swipe]'
    );
    sidebarSwipeEnabled.value =
      !noSideBarSwipe ||
      (noSideBarSwipe && noSideBarSwipe.getBoundingClientRect().x === 0);
```


**BEFORE:**

https://github.com/user-attachments/assets/93bea2f8-49fd-4f01-b1cd-ff8e4eeae415

**AFTER:**

https://github.com/user-attachments/assets/f921517a-df80-4ddb-8c57-27f1b582844f

Reply:
Weird, it works on your horizontal vote PR, but not here. `getBoundingClientRect()` always return `0`